packages/ods/src/components/input/tests/rendering/ods-input.e2e.ts

```diff
@@ -251,6 +251,29 @@ describe('ods-input rendering', () => {
 
       expect(buttonClearable).not.toBeNull();
     });
+
+    it('should render a clearable button when type', async() => {
+      await setup('<ods-input is-clearable type="number"></ods-input>');
+
+      await el.type('12');
+      await page.waitForChanges();
+      buttonClearable = await page.find('ods-input >>> ods-button[icon="xmark"]');
+      expect(buttonClearable).not.toBeNull();
+
+      await page.keyboard.press('Backspace');
+      await page.keyboard.press('Backspace');
+      await page.waitForChanges();
+
+      buttonClearable = await page.find('ods-input >>> ods-button[icon="xmark"]');
+      expect(buttonClearable).toBeNull();
+
+      // Test E2E launched with chrome. Chrome don't allow to type on input type number
```

Comment:
try typing "e" alone, it will be accepted by Chrome (as exponential symbol), but still considered an invalid value